snuba/api.py

```diff
@@ -412,14 +426,17 @@ def parse_and_run_query(dataset, validated_body, timer):
 @util.time_request('sdk-stats')
 def sdk_distribution(*, timer: Timer):
     body = parse_request_body(request)
-    validate_request_content(body, schemas.SDK_STATS_SCHEMA, timer)
 
-    body['project'] = []
-    body['aggregations'] = [
+    query, options = validate_request_content(body, schemas.SDK_STATS_QUERY_OPTIONS_SCHEMA, timer)
+    assert query['groupby'] == ['project_id']  # XXX: I hate this
```

Comment:
I probably should have deleted this comment but it's true. I want to get rid of this.